### DISCUSSION TITLE
Is there a way to customize a theme?

### DISCUSSION THREAD

**Discussion by MmAtBosch** [atom/atom#22722] [Q&A]
Hey,
i was not able to find an answer an this.

I like the One Dark theme. But i would like to have the foreground font color a bit whiter to add more contrast.
Is there any way to use a theme and change some settings of it?

Appreciate your answers.

**Comment by icecream17** [comment]
try editing your styles.less
https://flight-manual.atom.io/using-atom/sections/basic-customization/#style-tweaks
